src/components/question-wrapper.sass

```diff
@@ -2,7 +2,7 @@
 
 .questionWrapper
   .headers
-    height: 50px
+    height: 50px !important
```

Comment:
This rule and the rule on line 23 were being overruled by a generic style rule for DIVs in LARA 2 authoring forms (`height: auto`). I know relying on `!important` isn't a good practice generally, but it seemed like the most straightforward way to ensure that the elements keep their specified height. Since we're eventually going to restructure teacher edition authoring forms when there's more time, I thought this would be sufficient for now.